src/displayBackend/drm/Connector.cpp

```diff
@@ -91,10 +91,19 @@ void Connector::init(uint32_t width, uint32_t height,
 
 	if (mCrtcId != cInvalidId)
 	{
-		throw Exception("Already initialized", EINVAL);
+		throw
+		 Exception("Already initialized", EINVAL);
+	}
+
+	if (!frameBuffer) {
+		throw Exception("frameBuffer is nullptr. but must be initialized.", EINVAL);
```

Comment:
This is not in scope of current patch. Also, I believe this should be a plain `assert`